components/core/CMakeLists.txt

```diff
@@ -73,6 +73,22 @@ elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     endif ()
 endif ()
 
+# Java 11 is a dependency for ANTLR. The way we install Java
+# with brew doesn't also make it the default version of Java on the system. 
+# Set JAVA_HOME to Java 11 install location here.
+if (APPLE)
+    set(ENV{JAVA_HOME} "/usr/local/opt/openjdk@11/")
+endif ()
```

Comment:
Can we move this into `FindANTLR.cmake` so that we group the requirement for Java 11 with the `find_package(java...` call?